src/common/logging.cpp

```diff
@@ -54,12 +54,20 @@ static const char   sLevelString[][8] = {
       "[EMERG]", "[ALERT]", "[CRIT]", "[ERR ]", "[WARN]", "[NOTE]", "[INFO]", "[DEBG]",
 };
 
+static otbrLogLevel sDefaultLevel = OTBR_LOG_INFO;
```

Comment:
Do you know what is the current initial log level for OTBR on device? and align the value to the current initial level?

Reply:
The default value is INFO, and then it can be changed by command line argument to otbr-agent. We align this value with the otbr-agent initial value at line 88 of the current file: https://github.com/openthread/ot-br-posix/pull/1678/files#diff-ee208097f13a01b500e564c1b15f89e2184eaeb0b61f71f7bcbe684bc75709f8R88